.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules
 bower_components
+
```

Comment:
can we also add `.DS_Store` (and delete the file in this PR)

Reply:
done